recipe-app/src/pages/newForm/components/TextInput.jsx

```diff
@@ -6,7 +6,7 @@ function TextInput({ labelValue, type, name, id, onChangeHandler }) {
     <>
       <Label htmlFor={id}>{labelValue}</Label>
       <InputField
-        onChange={(e) => onChangeHandler(e)}
+        onChange={onChangeHandler}
```

Comment:
There is no need to express an anonymous function here. Just pass the onChangeHandler prop directly which will enable us to assign a handler function to the onChange event using the exposed onChangeHandler prop on the wrapper component i.e. <TextInput />